			     BASH PATCH REPORT
			     =================

Bash-Release:	4.3
Patch-ID:	bash43-044

Bug-Reported-by:	Ondrej Oprala <ooprala@redhat.com>
Bug-Reference-ID:	<539ED55B.2080103@redhat.com>
Bug-Reference-URL:	http://lists.gnu.org/archive/html/bug-bash/2014-06/msg00046.html

Bug-Description:

A typo prevents the `compat42' shopt option from working as intended.

Patch (apply with `patch -p0'):

--- a/builtins/shopt.def
+++ b/builtins/shopt.def
@@ -160,7 +160,7 @@ static struct {
   { "compat32", &shopt_compat32, set_compatibility_level },
   { "compat40", &shopt_compat40, set_compatibility_level },
   { "compat41", &shopt_compat41, set_compatibility_level },
-  { "compat42", &shopt_compat41, set_compatibility_level },
+  { "compat42", &shopt_compat42, set_compatibility_level },
 #if defined (READLINE)
   { "complete_fullquote", &complete_fullquote, (shopt_set_func_t *)NULL},
   { "direxpand", &dircomplete_expand, shopt_set_complete_direxpand },
--- a/patchlevel.h
+++ b/patchlevel.h
@@ -25,6 +25,6 @@
    regexp `^#define[ 	]*PATCHLEVEL', since that's what support/mkversion.sh
    looks for to find the patch level (for the sccs version string). */
 
-#define PATCHLEVEL 43
+#define PATCHLEVEL 44
 
 #endif /* _PATCHLEVEL_H_ */
